runtime/vm/createramclass.cpp

```diff
@@ -2663,7 +2772,10 @@ internalCreateRAMClassFromROMClassImpl(J9VMThread *vmThread, J9ClassLoader *clas
 
 	J9ROMClass *badClass = NULL;
 	bool incompatible = false;
-	if (!checkSuperClassAndInterfaces(vmThread, hostClassLoader, romClass, options, packageID, hotswapping, superclass, module, &badClass, &incompatible)
+	/* default value is -1, 0 is superclass, and 1 is superinterface */
```

Comment:
This comment is not clear to me. Is it for the value of `illegalAccessErrorTypes` ?

Reply:
Removed it.